src/test/java/de/tum/in/test/integration/testuser/subject/structural/AbstractClassExtension.java

```diff
@@ -0,0 +1,22 @@
+package de.tum.in.test.integration.testuser.subject.structural;
+
+public class AbstractClassExtension extends SomeAbstractClass {
```

Comment:
This is not used at the moment, do you plan to add tests here or can it be removed? (I think more tests would be better)